### PR TITLE
feat: clean up old design system `w-tooltip` class

### LINKED ISSUE #9711: Update all used `w-tooltip` to the new design system Tooltips
Implementation follow-up issue for https://github.com/GoogleChrome/web.dev/issues/9650.

Epic: https://github.com/GoogleChrome/web.dev/issues/9703

Update these templates from using `w-tooltip` (old design system) to new design system `tooltip`, following https://web.dev/design-system/component/tooltips/

- /lib/components/CopyCode/index.js
- /lib/components/Tooltip.js

**Note:** /src/styles/components/_tooltip.scss can be removed after refactoring those components.

### PR DESCRIPTION
**Pull request by mamieorine**
Fixes #9711 

Changes proposed in this pull request:

- Clean up old design system `w-tooltip` class and updated to new design system Tooltip.
- Remove a stylesheet for `w-tooltip`

### CODE REVIEW

**Review by jimjiminyjimjim — COMMENTED** [review]
LGTM

**Comment by mamieorine** [comment]
I made changes on `header-default.njk` and  `navigation-drawer-default.njk` since I spotted these templates are using old design system `w-tooltip`. However, those templates will be removed in [this PR](https://github.com/GoogleChrome/web.dev/pull/9720) and [9702](https://github.com/GoogleChrome/web.dev/issues/9702) anyway because they are unused templates.

File: src/lib/components/CopyCode/index.js [hunk 1]
```diff
@@ -22,17 +22,22 @@ class CopyCode extends BaseElement {
...
       // which means it isn't read reliably by screen readers.
       this.copyButton.setAttribute('aria-label', 'Copy code');
       this.copyButton.addEventListener('click', this.onCopy);
 
       this.tooltip = document.createElement('span');
-      this.tooltip.className = 'w-tooltip w-tooltip--right';
-      this.tooltip.setAttribute('role', 'tooltip');
-      this.tooltip.textContent = 'Copy code';
+      this.tooltip.className = 'tooltip';
+      this.tooltip.setAttribute('data-alignment', 'right');
```
Comment (matthiasrohmer) [inline src/lib/components/CopyCode/index.js:34]: I think we should keep `this.tooltip.setAttribute('role', 'tooltip');`. This is needed for proper a11y.

File: src/site/_includes/components/Tooltip.js [hunk 2]
```diff
@@ -18,20 +18,9 @@ const {html} = require('common-tags');
...
-      alignment = 'w-tooltip--right';
-      break;
-    default:
-      alignment = '';
-  }
-
   return html`
-    <span role="tooltip" class="w-tooltip ${alignment}">
-      ${md.renderInline(content)}
+    <span class="tooltip" data-alignment="${alignment}">
```
Comment (matthiasrohmer) [inline src/site/_includes/components/Tooltip.js:22]: Let's remove this component instead. I don't see the need for an author to use this in Markdown documents. Searching for `{% Tooltip` inside the codebase seems to show it also isn't used currently.

**Review by matthiasrohmer — CHANGES_REQUESTED** [review]

**Comment by mamieorine** [comment]
@matthiasrohmer  I have resolved all requested changes. Thank you!

**Review by matthiasrohmer — APPROVED** [review]

**Pull request merged**